mimoto-default.properties

```diff
@@ -35,7 +35,7 @@ mosip.event.secret=Kslk30SNF2AChs2
 
 
 mosip.partner.crypto.p12.filename=keystore.p12
-mosip.partner.crypto.p12.password={cipher}b77f8738b7fb8c48f84d587b045fa50099a569c381d1857eddbcd04afd83cd08
+mosip.partner.crypto.p12.password=abc123
```

Comment:
please use the below variable instead of hardcoding the password

`${mosip_partner_crypto_p12_password}`